ckanext/validation/templates/scheming/form_snippets/resource_schema.html

```diff
@@ -64,7 +64,6 @@
     {% set existing_value = h.scheming_display_json_value(value, indent=None) if is_json else value %}
     <input type="hidden" id="field-schema" name="schema" value="{{ existing_value }}" />
 
-    {% set type = 'asset' %}
-    {% include 'validation/snippets/validation_resource-schema-form_' ~ type ~ '.html' %}
+    {% asset 'ckanext-validation/resource-schema-form' %}
```

Comment:
Other templates still use `include`, why is this different?

Reply:
aligned to upstream

Reply:
This breaks `ckanext-data-qld` which still tries to `{% include 'validation/snippets/validation_resource-schema-form_' ~ type ~ '.html' %}`

Reply:
For backwards compatibility, can we update the syntax here but retain `validation_resource-schema-form_asset.html`?

Reply:
Raised https://github.com/qld-gov-au/ckanext-data-qld/pull/262 to adapt ckanext-data-qld to this.